code/lib/store/template/stories/rendering.stories.ts

```diff
@@ -40,21 +45,25 @@ export const ForceRemount = {
 export const ChangeArgs = {
   play: async ({ canvasElement, id }: PlayFunctionContext<any>) => {
     const channel = globalThis.__STORYBOOK_ADDONS_CHANNEL__;
+
+    await channel.emit(RESET_STORY_ARGS, { storyId: id });
+    await new Promise((resolve) => {
```

Comment:
Why do we need this? Doesn't seem like it was needed before?

Reply:
i just moved up we don't really need it i guess but it was last down in the function